packages/cw3/src/query.rs

```diff
@@ -42,32 +50,63 @@ pub enum Cw3QueryMsg {
+    /// small set of voters and participation is required. The advantage here is that if the
+    /// voting set (group) changes between proposals, the number of votes needed is adjusted
+    /// accordingly.
+    ///
+    /// Example: we set `percentage` to 51%. Proposal 1 starts when there is a `total_weight` of 5.
+    /// This will require 3 weight of yes votes in order to pass. Later, the Proposal 2 starts but the
+    /// `total_weight` of the group has increased to 9. That proposal will then automatically
+    /// require 5 yes of 9 to pass, rather than 3 yes of 9 as would be the case with `AbsoluteCount`.
     ///
-    /// Note: percentage_needed = 60% is different than threshold = 60%, quora = 100%
-    /// as the first will pass with 60% yes votes and 10% no votes, while the second
-    /// will require the others to vote anything (no, abstain...) to pass
+    /// A proposal of this type can pass early as soon as the needed weight of yes votes has been cast.
     AbsolutePercentage {
         percentage: Decimal,
         total_weight: u64,
     },
-    /// Declares a threshold (minimum percentage of votes that must approve)
-    /// and a quorum (minimum percentage of voter weight that must vote).
-    /// This allows eg. 25% of total weight YES to pass, if we have quorum of 40%
-    /// and threshold of 51% and most of the people sit out the election.
-    /// This is more common in general elections where participation is expected
-    /// to be low.
+
+    /// Declares a `quorum` of the total votes that must participate in the election in order
+    /// for the vote to be considered at all. Within the votes that were cast, it requires `threshold`
+    /// in favor. That is calculated by ignoring the abstain votes (they count towards `quorum`
+    /// but do not influence `threshold`). That is, we calculate `yes / (yes + no + veto)`
+    /// and compare that with `threshold` to consider if the proposal was passed.
+    ///
+    /// It is rather difficult for a proposal of this type to pass early. That can only happen if
+    /// the required quorum has been already met, and in the case if all remaining voters were
+    /// to vote no, the threshold would still be met.
+    ///
+    /// 30% yes votes, 10% no votes, and 20% abstain would pass early if quorum <= 60%
+    /// (who has cast votes) and if the threshold is <= 37.5% (the remaining 40% voting
+    /// no => 30% yes + 50% no). Once the voting period has passed with no additional votes,
+    /// that same proposal would be considered successful if quorum <= 60% and threshold <= 75%
+    /// (percent in favor if we ignore abstain votes).
+    ///
+    /// This type is more common in general elections where participation is expected to often
+    /// be low, and `AbsolutePercentage` would either be too restrictive to pass anything,
+    /// or allow low percentages to pass if there was high participation in one election.
```

Comment:
```suggestion
    /// In addition to a `threshold`, declares a `quorum` of the total votes that must participate in the election in order
    /// for the vote to be considered at all. Within the votes that were cast, it requires `threshold` votes
    /// in favor. That is calculated by ignoring the abstain votes (they count towards `quorum`
    /// but do not influence `threshold`). That is, we calculate `yes / (yes + no + veto)`
    /// and compare that with `threshold` to consider if the proposal was passed.
    ///
    /// It is rather difficult for a proposal of this type to pass early. That can only happen if
    /// the required quorum has been already met, and there are already enough Yes votes for the proposal to pass.
    ///
    /// 30% yes votes, 10% no votes, and 20% abstain would pass early if quorum <= 60%
    /// (who has cast votes) and if the threshold is <= 37.5% (the remaining 40% voting
    /// no => 30% yes + 50% no). Once the voting period has passed with no additional votes,
    /// that same proposal would be considered successful if quorum <= 60% and threshold <= 75%
    /// (percent in favor if we ignore abstain votes).
    ///
    /// This type is more common in general elections, where participation is often expected to
    /// be low, and `AbsolutePercentage` would either be too hight to pass anything,
    /// or allow low percentages to pass, independently of if there was high participation in one election or not.
```
Here, I think the example could be removed too.

We can still iterate on this a couple times IMO, as these are important / useful remarks. And also, in that way we make sure that we agree on them. ;-)

Reply:
I think the example is useful to explain why it is hard to pass early.
I could definitely be worded better